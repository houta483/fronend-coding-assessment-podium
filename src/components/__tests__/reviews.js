import React from "react"
import renderer from "react-test-renderer"
import ReactDOM from "react-dom"

import Reviews from "../reviews"

describe("Reviews", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Reviews siteTitle="Default Starter" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Reviews></Reviews>, div);
  ReactDOM.unmountComponentAtNode(div);
})



