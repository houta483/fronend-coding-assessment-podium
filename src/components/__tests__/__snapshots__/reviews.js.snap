// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews renders correctly 1`] = `
<div
  data-testid="parentDiv"
>
  <div
    className="flex-center red-text"
  >
    
  </div>
  <div
    className="flex-center bottom-padding"
  >
    <input
      onChange={[Function]}
      placeholder="Please enter a valid ID number"
      type="text"
    />
    <button
      onClick={[Function]}
      onPaste={[Function]}
    >
      Search
    </button>
  </div>
  <ul
    className="grid space-around"
  />
</div>
`;
